src/components/SocialBtns/SocialBtns.js

```diff
@@ -1,24 +1,32 @@
+/* global FB, location */
 import React, { PropTypes } from 'react';
 import FaFacebookSquare from 'react-icons/lib/fa/facebook-square';
 import FaTwitter from 'react-icons/lib/fa/twitter';
 import styles from './SocialBtns.sass';
 
 const SocialBtns = ({
-  fbLink = 'https://www.facebook.com',
-  twitterLink = 'https://twitter.com',
+  fbLink = `https://www.facebook.com/sharer/sharer.php?u=${location.href}`,
+  twitterLink = `https://twitter.com/intent/tweet?text=${location.href}`,
```

Comment:
Can we include the image, title and date? Right now its just a link that will be shortened by twitter's service with no other contextual information.